bochs/cpu/event.cc

```diff
@@ -129,6 +129,8 @@ void BX_CPU_C::InterruptAcknowledge(void)
   }
 #endif
 
+  BX_CPU_THIS_PTR EXT = 1; /* external event */
```

Comment:
I'm sorry but I couldn't understand why this line was moved above.

Reply:
Actually no particular reason. This has no change at all.
Originally I have rewritten this block completely but undone the change.
But seems like undone not 100% exactly